charts/harvester-csi-driver/Chart.yaml

```diff
@@ -33,6 +33,7 @@ annotations:
   catalog.cattle.io/namespace: kube-system
   catalog.cattle.io/release-name: harvester-csi-driver
   catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux
```

Comment:
will this annotation be required for all future charts submitted against the rancher/charts?

Reply:
Not sure.